### DISCUSSION TITLE
Please reissue license

### DISCUSSION THREAD

**Discussion by smkurian1972** [aclap-dev/video-downloadhelper#285] [Help and Q&A]
### Your question

Hi. I paid for a license but cannot  locate the email with the license for premium downloads. Email is smkurian@hotmail.com. Thank you for your help. 

### Addon & CoApp version and details

version: 8.2.1.1
channel: stable
build date: 2024-02-19
build options: linuxlic: false, noyt: true, browser: chrome
lang: en-US
license: unset
platform: x86-64 win
UA: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/121.0.0.0 Safari/537.36
coapp: yes. Info:
{
  "id": "net.downloadhelper.coapp",
  "name": "VdhCoApp",
  "version": "2.0.17",
  "binary": "C:\\Program Files\\DownloadHelper CoApp\\vdhcoapp.exe",
  "displayName": "VdhCoApp",
  "description": "Video DownloadHelper companion app",
  "target": {
    "os": "windows",
    "arch": "x86_64",
    "node": "18"
  },
  "home": "C:\\Users\\mikes",
  "converterBinary": "C:\\Program Files\\DownloadHelper CoApp\\ffmpeg.exe",
  "converterBase": "ffmpeg",
  "converterBaseVersion": "c30f360955"
}

**Comment by jcfrog** [comment]
Hi,
I sent you an email.

**Comment by smkurian1972** [comment]
Thank you!


Sunil Kurian, PhD

Research Scientist,

Scripps Clinic Bio-Repository & Bio-Informatics Core

Scripps Center for Organ & Cell Transplantation Research



Linkedin: https://www.linkedin.com/in/sunil-kurian-4043174/



Chief Scientific Advisor

MindX Sciences

https://mindxsciences.com/



Scientific Advisor

Transplant Genomics Inc.,

http://transplantgenomics.com/advisors/

________________________________
